README.md

```diff
@@ -57,6 +57,7 @@ $ yarn service:timeseriesWithoutAggregateMigration
 
 `coachco2.admin-mode`: activate some hidden functions
 `coachco2.dacc-dev_v2`: to use dev version of DACC
+`coachco2.bikegoal.enabled`: to use dev version of objectives
```

Comment:
`To activate the "bike goal" feature`

on n'utilise plus le mot clé "objectives" et il ne s'agit pas de version de dev ici